code/modules/admin/verbs/adminhelp.dm

```diff
@@ -369,9 +369,10 @@ GLOBAL_DATUM_INIT(ahelp_tickets, /datum/admin_help_tickets, new)
 	if(state != AHELP_ACTIVE)
 		return
 
-	var/msg = "<font color='red' size='4'><b>- AdminHelp marked as IC issue! -</b></font><br>"
-	msg += "<font color='red'><b>Losing is part of the game!</b></font><br>"
-	msg += "<font color='red'>It is also possible that your ahelp is unable to be answered properly, due to events occurring in the round.</font>"
+	var/msg = "<font color='red' size='4'><b>- Your AdminHelp has been marked as IC issue! -</b></font><br>"
+	msg += "<font color='red'><b>An admin has carefully looked at the submitted issue and has determined that the reported incident is valid.</b></font><br>"
+	msg += "<font color='red'>Some actions by users, while appearing malicious, can actually be legitimate due to their status or given situation in the round.</font>"
+	msg += "<font color='red'>Please do not be discouraged from reporting similar instances in the future. While not all actions are invalid, not all actions are valid.</font>"

```

Comment:
Lol, imagine unironically turning the dedicated "grow a pair, dude" button into an overly patronizing "uwu its valid uwu uwu uwu" button
The "Resolve" and "Close" buttons are for ahelps about IC issues that don't exactly warrant telling someone to grow a pair. The IC issue button is for when someone is extra salty about dying via valid cause or otherwise being inconvenienced.

Reply:
imagine having this shit-tier attitude. there is literally no reason to get this hostile over a pr lmao but sure.

The grow of pair button is no longer used as a grow a pair button but a quick and easy way of telling someone it's an IC issue regardless of how stupid the ahelp is. In 100% of instances where I ahelped shit that looked like bad behavior but I wasn't actually affected by the actions of the person, the button was used and there is nothing wrong with that. Literally everyone I talked to said the same thing and it's not because they are a part of the group of a "certain type of people" and the admins I talked to say they don't use it to tell people to grow a pair but to tell them that it's an IC issue without specifically saying "yeah that dude is an antag." or leaking any information about the round.

It's a useful button for admins dealing with a lot of shit and there is literally nothing wrong with that. Everyone knows that its the purpose. Admins don't have time to tell everyone in someone who died in a random explosion to kill one target that it was valid. Admins don't have time to tell everyone affected in a plasma flood 5 minutes into the shift that it's valid. If admins want to tell someone to grow a pair they literally use the "man up button" but I guess we're pretending that doesn't exist?

This is especially useful for new players and salty players. As I explained in the staff forum it's super useful for salty players to read and understand because it gives them more to chew on. It explains the purpose of the button and tells them to relax without specifically telling them to relax instead of just the "dying is a part of the game" which literally everyone and their dog knows.